certify-service/src/main/java/io/mosip/certify/proofgenerators/Ed25519Signature2020ProofGenerator.java

```diff
@@ -10,9 +10,12 @@
 import io.mosip.kernel.signature.dto.SignResponseDto;
 import io.mosip.kernel.signature.service.SignatureServicev2;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
```

Comment:
@vishwa-vyom Even when `VCIssuance` is configured, this bean would have to be created, in this case, would it benefit us if we specify this as a `ConditionalOnMissingBean` ?